index.html

```diff
@@ -46,181 +56,249 @@ <h2 class="sub-title">I'm a Software Developer</h2>
-                        <li><button class="html-c" type="button">HTML</button></li>
-                        <li><button class="bootstrap-c" type="button">Bootstrap</button></li>
-                        <li><button class="ruby-c" type="button">Ruby</button></li>
-                    </ul>
-                    <button class="s-button" type="button">See project</button>
-                </article>
-                <article class="art-data">
-                    <img class="bg-img2" src="assets/recent_works/bg-img.png" alt="background-image"/>
-                    <div class="overlay-content">
-                        <h3 class="title-post">Professional Art <br> Printing Data</h3>
-                        <p class="description">
-                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
-                        </p>
-                        <ul class="stack-2">
-                            <li><button class="html" type="button">HTML</button></li>
-                            <li><button class="bootstrap" type="button">Bootstrap</button></li>
-                            <li><button class="ruby" type="button">Ruby</button></li>
-                        </ul>
+                    <div class="recent-works">
+                        <h2 class="title">My Recent Works</h2>
+                        <hr class="custom-hr">
                     </div>
-                    <div class="grid-container">
-                        <div class="grid-item">
-                            <button class="b-button" type="button">See project</button>
+                    <div class="recent-projects">
+                        <img class="custom-img-mobile" src="assets/recent_works/hm_img.png" alt="placeholder img"/>
+                        <img class="custom-img-desktop" src="assets/recent_works/bg-img-desktop.png" alt="placeholder img"/>
+                        <div class="stories">
+                            <h3 class="s-title">Multi-Post Stories</h3>
+                            <p class="paragraph">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                            <ul class="stack">
+                                <li><button class="css-c" type="button">CSS</button></li>
+                                <li><button class="html-c" type="button">HTML</button></li>
+                                <li><button class="bootstrap-c" type="button">Bootstrap</button></li>
+                                <li><button class="ruby-c" type="button">Ruby</button></li>
+                            </ul>
+                            <button class="s-button" type="button">See project</button>
```

Comment:
Kindly add some margin-top to this section of your project, so it doesn't look clustered 😄 

Your design :
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/20537712/8b3c6bde-1ec5-49fa-bdd2-619911545337)